metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -0,0 +1,26 @@
+=== windows perfmon MetricSet
+
+This is the perfmon metricset of the module windows.
+
+[float]
+=== Features and configuration
+
+Metricset to collect performance counters.
+Example configuration:
+```
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 10s
+  perfmon.counters:
+    - group: "processor"  
+      collectors:
+        - alias: "processor_perfomance"
```

Comment:
As far as I understand the code, the group name is only needed to create the Event but not for the API request to fetch the event. If you use `common.MapStr.Put("my.key.is.nested")` you will also get a nested object. 

So my question is if it would also work, if the config would look as following:

```
- module: windows
  metricsets: ["perfmon"]
  enabled: true
  period: 10s
  perfmon.counters:
    - alias: "processor_perfomance"
      query: '\Processor Information(_Total)\% Processor Performance'
    - alias: "processor_time"
      query: "\\Processor Information(_Total)\\% Performance Limit"
    - alias: "disk.bytes_written"
      query: '\FileSystem Disk Activity(_Total)\FileSystem Bytes Written'
```